ui/src/app/views/reset-cluster-button/component.tsx

```diff
@@ -84,12 +82,14 @@ export class SettingsReact extends React.Component<{}, SettingsState> {
              loading={this.state.resetClusterPending}>
         <ModalTitle>Reset Notebook Server?</ModalTitle>
         <ModalBody>
-          <strong>Warning:</strong> Any unsaved changes to your notebooks may be lost
-          and your cluster will be offline for 5-10 minutes.
-          <br/><br/>
-          Resetting should not be necessary under normal conditions. Please help us to
-          improve this experience by using "Report a Bug" from the profile drop-down
-          to describe the reason for this reset.
+            <div>
+              <strong>Warning:</strong> Any unsaved changes to your notebooks may be lost
+              and your cluster will be offline for 5-10 minutes.
+              <br/><br/>
+              Resetting should not be necessary under normal conditions. Please help us to
+              improve this experience by using "Report a Bug" from the profile drop-down
```

Comment:
this text should change to refer users to "Contact Support" from the side hamburger menu.  "Report a Bug" on the profile drop down no longer exists since we released zendesk.